mrjob/spark/mrjob_spark_harness.py

```diff
@@ -72,17 +72,43 @@ def _run_step(step, step_num, rdd, make_job):
             lambda pairs: mapper_job.map_pairs(pairs, step_num))
         rdd = rdd.map(lambda k_v: m_write(*k_v))
 
+    if step_desc.get('combiner'):
+        combiner_job = make_job('--combiner', '--step-num=%d' % step_num)
+        c_read, c_write = combiner_job.pick_protocols(step_num, 'combiner')
+
+        # run the combiner
+        rdd = rdd.map(c_read)
+
+        def combiner_helper(pairs1, pairs2):
```

Comment:
This would be a good place for a comment about why we expect a single item list to be the common case, and why we don't both trying to combine lists with multiple values.